lib/config/options/index.ts

```diff
@@ -1247,6 +1247,20 @@ const options: RenovateOptions[] = [
     cli: false,
     mergeable: true,
   },
+  {
+    name: 'pinDigest',
+    description:
+      'Configuration to apply when pinning a digest (no change in tag/version).',
+    stage: 'package',
+    type: 'object',
+    default: {
+      branchTopic: '{{{depNameSanitized}}}-pin-digest',
```

Comment:
@rarkins How about we set the pr title for this case to be :
```suggestion
      branchTopic: '{{{depNameSanitized}}}-pin-digest',
      prTitle:'Pin {{{depName}}} digest to {{{newDigestShort}}}',
```